meta-refkit/conf/distro/include/refkit-ci.inc

```diff
@@ -49,6 +49,11 @@ require conf/distro/include/refkit-development.inc
 REFKIT_VM_IMAGE_TYPES = "wic.xz wic.zip wic.bmap wic.xz.sha256sum"
 
 #
+# pre/post-build oe-selftests started by CI builder, whitespace-separated.
```

Comment:
I added one more comment line there.
Does the current version address all change requests for now?